src/overlays/effects/ovl_Effect_Ss_Lightning/z_eff_ss_lightning.c

```diff
@@ -1,29 +1,157 @@
 /*
  * File: z_eff_ss_lightning.c
  * Overlay: ovl_Effect_Ss_Lightning
- * Description:
+ * Description: Lightning
```

Comment:
As in electrical arcs, presumably, rather than weather? I think "lightning" is usually reserved for cloud-based arcs.
> Lightning is a naturally occurring [electrostatic discharge](https://en.wikipedia.org/wiki/Electrostatic_discharge) during which two [electrically charged](https://en.wikipedia.org/wiki/Electric_charge) regions, both in the [atmosphere](https://en.wikipedia.org/wiki/Atmosphere) or with one on the ground, temporarily neutralize themselves,

(https://en.wikipedia.org/wiki/Lightning)

(Yes I know it's in the name, but better to be more precise if possible, I think.)

Reply:
Fair point. Probably better to just go with the general `Electric Discharge` since it does have nothing to do with weather. (It also doesn't look to be used in MM, just OOT leftover for specific enemies such as the jellyfish).